### PR TITLE
Fix the example and make it docspec in Streamly.Network.Socket

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

**Comment by adithyaov** [comment]
The haddock isn't displayed properly.

Screenshot from 2022-10-27 15-42-55

**Comment by harendra-kumar** [comment]
Can you raise an issue in the haddock repo for the haddock indentation issue?

File: src/Streamly/Network/Socket.hs [hunk 1]
```diff
@@ -27,38 +27,41 @@
...
+-- >>> import Network.Socket
+-- >>> import Streamly.Network.Socket (SockSpec(..))
+-- >>>
+-- >>> import qualified Streamly.Data.Fold as Fold
+-- >>> import qualified Streamly.Data.Stream as Stream
+-- >>> import qualified Streamly.Data.Stream.Concurrent as Concur
+-- >>> import qualified Streamly.Network.Socket as Socket
+-- >>>
+-- >>> :{
+-- >>> main :: IO ()
```
Comment (harendra-kumar) [inline src/Streamly/Network/Socket.hs:43]: You need to remove the `>>>` inside the block `:{` `:}`.

**Comment by harendra-kumar** [comment]
> Can you raise an issue in the haddock repo for the haddock indentation issue? But I see that haddock is being displayed properly in other cases.

Ok, this is not a haddock issue. You need to correct the formatting, remove `>>>` inside multiline blocks.

File: src/Streamly/Network/Socket.hs (hunk 1 above)
Comment (adithyaov) [reply src/Streamly/Network/Socket.hs:43]: Ah, gotcha!

**Review by harendra-kumar — APPROVED** [review]

**Pull request merged**